data/chiragjh7.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Chirag Jhariya",
+  "displayStatsPublic": true,
```

Comment:
We've recently removed the public stats feature, so `displayStatsPublic` property is no longer needed. 
Please remove this from your json. 

```suggestion
```